### PR TITLE
ER-316 successful application

### PR DESCRIPTION
**Pull request by chrisjensenuk**
- Added command `ApplicationReviewSuccessfulCommand`
- Added command handler `ApplicationReviewCommandHandler` to handle application review commands
- Added event `ApplicationReviewSuccessfulEvent`

After setting application review to 'Successful' an `ApplicationReviewSuccessfulEvent` is published.
This event is handled by:
- `UpdateDashboardOnVacancyChange` to rebuild the dashboard query view (there are new application counts against the vacancy that need updating)
- `UpdateVacancyApplicationsOnApplicationReviewChange` - to rebuild the applications against the vacancy
- In a future ticket we'll add an event handler to push a message to FAA to update it's application

### CODE REVIEW

File: src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml [hunk 1]
```diff
@@ -253,21 +225,18 @@
...
-                </div>
 
-            </div>
+        <section>
+            <h2 class="heading-large">Skills the Employer wants</h2>
 
             <p asp-condition="@Model.HasNoSkills">I don't have the skills the employer wants</p>
 
             <p asp-condition="@Model.HasSkills">
                 @Html.Raw(string.Join(", ", Model.Skills.Select(s => string.Format("<span>{0}</span>", Html.Encode(s)))))
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml]: Do we have to surround each skill in a `span`? 👍 for the encoding, I think we should do this on the preview views as well.

File: src/Employer/Employer.Web/Extensions/ApplicationReviewExtensions.cs [hunk 2]
```diff
@@ -35,6 +35,7 @@ public static ApplicationReviewViewModel ToViewModel(this ApplicationReview r)
                 EducationToYear = r.Application.EducationToYear,
                 HobbiesAndInterests = r.Application.HobbiesAndInterests,
                 Improvements = r.Application.Improvements,
+                Outcome = r.Status,
                 Phone = r.Application.Phone,
                 Qualifcations = r.Application.Qualifications?.Select(q =>
```
Comment (sachpatel) [inline src/Employer/Employer.Web/Extensions/ApplicationReviewExtensions.cs]: Qualifications

File: src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml (hunk 1 above)
Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Views/ApplicationReview/ApplicationReview.cshtml]: 👍

File: src/Employer/Employer.Web/Extensions/ApplicationReviewExtensions.cs (hunk 2 above)
Comment (chrisjensenuk) [reply src/Employer/Employer.Web/Extensions/ApplicationReviewExtensions.cs]: 👍

**Pull request merged**